Neos.Fusion/Classes/Core/Cache/ParserCacheFlusher.php

```diff
@@ -50,7 +50,10 @@ public function flushPartialCacheOnFileChanges($fileMonitorIdentifier, array $ch
 
         $identifiersToFlush = [];
         foreach ($changedFiles as $changedFile => $status) {
-            $identifiersToFlush[] = $this->getCacheIdentifierForFile($changedFile);
+            // flow already returns absolute file paths from the file monitor, so we don't have to call realpath.
```

Comment:
That is an assumption that could change, no? I check here might make sense? Why not just check the result of `realpath()`?

Reply:
At least flow has been that way since file monitors exist. I also talked with @kitsunet and we agreed that this is a fair assumption.